controller/command/reconciler.go

```diff
@@ -61,6 +61,17 @@ func (r *Reconciler) eval() {
 }
 
 func (r *Reconciler) sync() {
+	// Check for deletion timestamp on command resource
+	// if set then command is marked for deletion
+	if !r.observedCommand.DeletionTimestamp.IsZero() {
+		klog.V(1).Infof(
+			"Will skip command reconciliation: It is marked for deletion: Command %q / %q",
+			r.observedCommand.GetNamespace(),
+			r.observedCommand.GetName(),
+		)
+		return
```

Comment:
There is a specific variable that can be set to skip reconcile.
Lets set that.
Please check the embedded reconciler properties & logic as well.

Reply:
I don't think it will stop reconciliation on the command but it says [skip](https://github.com/AmitKumarDas/metac/blob/master/controller/generic/hooks.go#L94-L96) reconciliation on attachments

Reply:
We are already setting it https://github.com/mayadata-io/d-operators/blob/master/controller/command/reconciler.go#L123-L125

Reply:
ok. perfect.